pkgs/applications/misc/scim/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, pkgs, ncurses, libxml2 }:
+
+let
+  version = "0.1.8";
+in
+stdenv.mkDerivation rec {
+
+  name = "scim-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/andmarti1424/scim/archive/v${version}.tar.gz";
+    sha256 = "0qjixb1hzbdrypbmzwb2iaw5wp57kn7fmm1zpjp4gzjyanrhazs2";
+  };
+
+  buildInputs = with pkgs; [ yacc ncurses libxml2 ];
+
+  buildPhase = ''
+    cd src
+
+    sed "s,prefix=/usr,prefix=$out," Makefile
```

Comment:
But I need the `cd src` in the buildPhase anyways... so I guess there is no point splitting it, is there?